source/parser.hera

```diff
@@ -360,7 +360,7 @@ ExtendsToken
     return { $loc, token: $1 }
 
 ExtendsTarget
-  LeftHandSideExpression TypeArguments?
```

Comment:
I think this one should stay `TypeArguments` and `TypeArguments` should have `{ts: true, children: $0}` added. The `TypeParameters` rule also probably needs to be updated because it should only allow type identifiers and not any type.

The thinking is that TypeParameters are like function parameters in that they name a slot for a type to be received and  that TypeArguments are the actual types being slotted in.

Reply:
Ah ok, so in following example: 

```ts
type Foo<T> = { x: T }
                ^
                 Type parameter

type FooStr = Foo<String>
                               ^
                                Type arguments
```

Then type parameters will need to be restricted to identifiers (and optionally constraints) where as Type args will be concrete types.